src/BenchmarkDotNet/Environments/Runtimes/WasmRuntime.cs

```diff
@@ -17,6 +17,10 @@ public class WasmRuntime : Runtime, IEquatable<WasmRuntime>
 
         public string JavaScriptEngineArguments { get;  }
 
+        public bool Aot { get;  }
+
+        public DirectoryInfo RuntimeSrcDir { get;  }
```

Comment:
this type implements `IEquatable<T>` so if you add new properties you should also modify `Equals` and `GetHashCode`